### ISSUE TITLE
[Suggestion]Wing block for simple lift in ring atmosphere or future planets

### ISSUE THREAD

**Issue by MartinF20M** [tsunamayo/Starship-EVO#5029]
As to title says Ive been working on a plane and would love to see some sort of a wing block, as i know aerodynamics is extremely hard to implement, maybe something like a wing block would just provide lift, more wings plus more speed equals more lift no need for upwards thrust something like that in empyrion or that wing block mod for space engineers, obviously not that important right now but maybe for the future

**Comment by tsunamayo** [comment]
Yes I would like to offer a way to provide lift, but more by looking at the shape of the build rather than through special blocks. Cheers

**Comment by Briaireous** [comment]
> Yes I would like to offer a way to provide lift, but more by looking at the shape of the build rather than through special blocks. Cheers

FUKIN YES! YES THAT WOULD BE AWESOME
